NitroxLauncher/OptionPage.xaml

```diff
@@ -4,40 +4,46 @@
       xmlns:d="http://schemas.microsoft.com/expression/blend/2008" 
       xmlns:local="clr-namespace:NitroxLauncher"
-      mc:Ignorable="d" 
-      d:DesignHeight="470" d:DesignWidth="719"
+      mc:Ignorable="d" d:DesignHeight="470" d:DesignWidth="719"
       DataContext="{Binding RelativeSource={RelativeSource Self}}"
-      Title="OptionPage"
-      Name="OptionsPage" 
+      Title="{DynamicResource ResourceKey=NitroxOptionTitle}"
       Margin="0">
-    <Page.Resources>
-        
-    </Page.Resources>
+
     <Grid >
         <Image x:Name="BackgroundImage" Source="Images/Vines.png" Grid.Column="0" Grid.ColumnSpan="2" Grid.RowSpan="2" Height="553" Width="1125.43" Opacity="100" HorizontalAlignment="Right" VerticalAlignment="Bottom"/>
-        <Grid Height="470" Width="719" Margin="30,0,0,0">
+        <Grid Margin="25,0,0,0">
             <Grid.RowDefinitions>
+                <RowDefinition Height="30"/>
+                <RowDefinition Height="Auto"/>
                 <RowDefinition Height="Auto"/>
                 <RowDefinition Height="Auto"/>
                 <RowDefinition Height="Auto"/>
                 <RowDefinition Height="*"/>
             </Grid.RowDefinitions>
 
-            <Grid Grid.Row="0" Height="32"></Grid>
             <Grid Grid.Row="1">
-                <TextBlock FontSize="22" Foreground="White">Options</TextBlock>
+                <TextBlock FontSize="22" Foreground="White" Text="{DynamicResource ResourceKey=NitroxOTitle}"/>
             </Grid>
+
             <StackPanel Grid.Row="2" Margin="0,17,0,0">
-                <TextBlock FontFamily="{StaticResource Roboto}" FontSize="12" Foreground="White">GAME DIRECTORY</TextBlock>
-                <Border  HorizontalAlignment="Left" Height="43" BorderBrush="#33FFFFFF" Background="#9928292C" BorderThickness="1" CornerRadius="4,4,4,4" Margin="0,4,0,0">
-                    <TextBox FontFamily="{StaticResource Roboto}" Width="427" Height="43" FontSize="16" MaxLines="1" Foreground="White" TextAlignment="Left" Padding="9" Background="{x:Null}" BorderBrush="{x:Null}" Text="{Binding PathToSubnautica, RelativeSource={RelativeSource FindAncestor, AncestorType=Page}}" SelectionOpacity="-1" ToolTip="Subnautica install location"/>
+                <TextBlock FontFamily="{StaticResource Roboto}" FontSize="12" Foreground="White" Text="{DynamicResource ResourceKey=NitroxOptionGameDirectory}"/>
+                <Border HorizontalAlignment="Left" Height="43" BorderBrush="#33FFFFFF" Background="#9928292C" BorderThickness="1" CornerRadius="4,4,4,4" Margin="0,4,0,0">
+                    <TextBox IsReadOnly="True" FontFamily="{StaticResource Roboto}" Width="427" Height="43" FontSize="16" MaxLines="1" Foreground="White" TextAlignment="Left" Padding="9" Background="{x:Null}" BorderBrush="{x:Null}" Text="{Binding PathToSubnautica, Mode=OneWay}" SelectionOpacity="-1" ToolTip="{DynamicResource ResourceKey=NitroxOptionGameDirectoryTooltip}" />
```

Comment:
Why is this converted to readonly/OneWay? Users should be able to change the path, especially when the default is chosen because no game is found using the likely limited detection methods.

Reply:
Because the textbox isn't implemented in TwoWay so the textbox is only used to display the value of the path when you change throught the button